### ISSUE TITLE
Add accessibility documentation

### ISSUE THREAD

**Issue by mvogelgesang** [GSA/uswds-sf-lightning-community#180]
**Is your feature request related to a problem? Please describe.**

Idea from @freemacj Provide greater detail for implementers about what accessibility testing took place, known issues, etc

**Describe the solution you'd like**

- Specify a standard process to test for accessibility. This should include the tools used, compliance criteria, reporting mechanisms for identified issues
  - [x] This could be accomplished by testing each component as an individual page in the Demo Community, #111 . Using [Lighthouse](https://www.npmjs.com/package/lighthouse#using-the-node-cli), each URL can be evaluated and an HTML report generated. `npx lighthouse https://inspiration-ability-2765.scratch.my.site.com/issue111/s/ --only-categories=accessibility --view --output html`
  - [x] Write up this process in the wiki or contributing guide
- [x] Add validation to components to ensure alt text is present.
- Specify a process to share the results of these tests
  - [x] Before merging new components or updates to components, add step to test using Lighthouse
  - [ ] Update release process to incorporate the sharing of test results for each component

**Comment by mvogelgesang** [comment]
Ideally, accessibility tests for each component would be performed in isolation. Looking at the various community types, this does not appear to be doable as header and footer components would always be present. Lighthouse tests were run against the new demo community #111 and the relevant pages there. No issues arose for the components present in the demo community. 

Separately, USWDS components that leverage images have validation built in which will call out a lack of alt text and direct them to resources on the topic. This does not prevent someone from entering poor alt text but does aim to encourage better practices. For example, USWDS Hero throws a Builder Notification stating "Alt Text is missing for the background image. Here is more info about how to write great alt text, https://www.w3.org/TR/WCAG20-TECHS/H37.html"
